internal/nodejs/plan.go

```diff
@@ -411,6 +416,7 @@ func GetStaticOutputDir(ctx *nodePlanContext) string {
 		types.NodeProjectFrameworkVitepress:      "docs/.vitepress/dist",
 		types.NodeProjectFrameworkAstroStatic:    "dist",
 		types.NodeProjectFrameworkSliDev:         "dist",
+		types.NodeProjectFrameworkDocusaurus:     "build",
```

Comment:
Is this output directory fixed? (users can't change it in an easy way)

Reply:
@pan93412 This should be the **default** output directory of the framework. If user change the config, they can pass the custom output path from `buildOptions`.